src/Akka.Streams.Kafka/reference.conf

```diff
@@ -13,6 +13,11 @@ akka.kafka.producer {
   # When this value is empty, the dispatcher configured for the stream
   # will be used.
   use-dispatcher = "akka.kafka.default-dispatcher"
+  
+  # Properties defined by Confluent.Kafka.ProducerConfig
+  # can be defined in this configuration section.
+  kafka-clients {
+  }
```

Comment:
This is where Kafka properties can be declared directly inside HOCON

Reply:
nice - might need to add some docs about this and about being able to consume the `ConsumerConfig` and `ProducerConfig` sections directly in the builder.